packages/config/src/chainConfig/configs/mainnet.ts

```diff
@@ -52,6 +52,10 @@ export const chainConfig: ChainConfig = {
   ELECTRA_FORK_VERSION: b("0x05000000"),
   ELECTRA_FORK_EPOCH: Infinity,
 
+  // FULU
+  FULU_FORK_VERSION: b("0x06000000"),
```

Comment:
I think it would be good practice to add this to all network configs right away, see https://github.com/ChainSafe/lodestar/pull/7401 for reasoning